pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -182,11 +182,7 @@ func Pod(source *operatorsv1alpha1.CatalogSource, name string, image string, saN
 						},
 					},
 					SecurityContext: &corev1.SecurityContext{
-						ReadOnlyRootFilesystem:   pointer.Bool(false),
-						AllowPrivilegeEscalation: pointer.Bool(false),
-						Capabilities: &corev1.Capabilities{
-							Drop: []corev1.Capability{"ALL"},
-						},
+						ReadOnlyRootFilesystem: pointer.Bool(false),
```

Comment:
Do "restricted" mode catalogs actually need a writeable root filesystem?

Reply:
I'm guessing "restricted" mode doesn't change the fact the [database might have to be copied to a different location in a file system ](https://github.com/operator-framework/operator-lifecycle-manager/pull/2167). Messing with this sounds like a potential to reintroduce a bug that PR was trying to solve for.

Reply:
> I'm guessing "restricted" mode doesn't change the fact the https://github.com/operator-framework/operator-lifecycle-manager/pull/2167

I was thinking we copy to /tmp now and that /tmp might be mounted separately, but looking at a catalog pod it appears /tmp is part of the root filesystem, so i guess not (even if we are copying to /tmp)

So yeah, we can close this comment.

Reply:
For sqlite, yes. opm will still create the database copy (but in the /tmp directory now =D)